### PR TITLE
Change action name from "Skip All Breakpoints" to "Toggle Skip All Breakpoints"

### PR DESCRIPTION
**Pull request by ShahzaibIbrahim**
Change title from "Skip All Breakpoints" to "Toggle Skip All Breakpoints" when toggled

### CODE REVIEW

**Review by HeikoKlare — COMMENTED** [review]
Thanks for the proposal! It makes sense to have all toggle options uniformly prefixed with "toggle" to make the effect easier to understand.

Here are some remarks with respect to this change:
1. There are some futher NLSs for the same action that may need to be adapted (see detailed comments).
2. With this change, the names of the actions within the `Breakpoints` view become inconsistent. There are other toggleable options, such as "Show Breakpoints Supported by Selected Target", which also miss a "Toggle" prefix.
image
3. To me "Toggle Skip All Breakpoints" sounds a bit odd, because in contrast to all other "Toggle X" actions, the "X" is not a noun here (like in "Toggle Breakpoint") but a verb-based action description. Unfortunately, I do not have a really good alternative either. "Toggle Skipping All Breakpoints" is probably not much better...
4. There seems to be explicit documentation for this action, which probably has to be updated when changing the action name, e.g.: https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/blob/4b5fccff1c7878e02430c22ede26c8788c60896e/eclipse.platform.common/bundles/org.eclipse.jdt.doc.user/reference/views/breakpoints/ref-skipall_viewaction.htm#L8

**Comment by ShahzaibIbrahim** [comment]
> Thanks for the proposal! It makes sense to have all toggle options uniformly prefixed with "toggle" to make the effect easier to understand.
> 
> Here are some remarks with respect to this change:
> 
> 1. There are some futher NLSs for the same action that may need to be adapted (see detailed comments).
> 2. With this change, the names of the actions within the `Breakpoints` view become inconsistent. There are other toggleable options, such as "Show Breakpoints Supported by Selected Target", which also miss a "Toggle" prefix.
>    image
> 3. To me "Toggle Skip All Breakpoints" sounds a bit odd, because in contrast to all other "Toggle X" actions, the "X" is not a noun here (like in "Toggle Breakpoint") but a verb-based action description. Unfortunately, I do not have a really good alternative either. "Toggle Skipping All Breakpoints" is probably not much better...
> 4. There seems to be explicit documentation for this action, which probably has to be updated when changing the action name, e.g.: https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/blob/4b5fccff1c7878e02430c22ede26c8788c60896e/eclipse.platform.common/bundles/org.eclipse.jdt.doc.user/reference/views/breakpoints/ref-skipall_viewaction.htm#L8

Understood. If you prefer to keep a consistent pattern with "Toggle X," you might consider using a noun that represents the action of skipping breakpoints. Here is a suggestion:

"Toggle Breakpoint Skipping"

File: debug/org.eclipse.debug.ui/plugin.properties [hunk 1]
```diff
@@ -280,7 +280,7 @@ ActionDefinition.removeAllBreakpoints.description=Removes all breakpoints
 ActionDefinition.skipAllBreakpoints.name=Skip All Breakpoints
```
[the remark this answers is not in the document]
Comment (ShahzaibIbrahim) [reply debug/org.eclipse.debug.ui/plugin.properties:280]: I have to check the usage for it.

**Comment by ShahzaibIbrahim** [comment]
> 4. There seems to be explicit documentation for this action, which probably has to be updated when changing the action name, e.g.: https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/blob/4b5fccff1c7878e02430c22ede26c8788c60896e/eclipse.platform.common/bundles/org.eclipse.jdt.doc.user/reference/views/breakpoints/ref-skipall_viewaction.htm#L8

https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/1559

**Comment by vogella** [comment]
@HeikoKlare anything left here to do for @ShahzaibIbrahim?

**Comment by ShahzaibIbrahim** [comment]
> @HeikoKlare anything left here to do for @ShahzaibIbrahim?

@vogella Yes, I have found the existing ticket for this particular issue (See: https://github.com/eclipse-platform/eclipse.platform.swt/issues/501) and the toggle "state/highlight" issue is for windows 11 only. I have one more suggestion in mind which I will propose soon, till then we can draft this PR.

**Pull request closed without merging**